pkg/verifiedcar/verifiedcar.go

```diff
@@ -53,6 +53,7 @@ type Config struct {
 	ExpectDuplicatesIn bool           // Handles whether the incoming stream has duplicates
 	WriteDuplicatesOut bool           // Handles whether duplicates should be written a second time as blocks
 	MaxBlocks          uint64         // set a budget for the traversal
+	OnRemoteBytesRead  func(uint64)   // a callback whenever bytes are read from the stream
```

Comment:
This could just be `OnBlockRead`; thinking with a lens of this code being pulled out and used independent of Lassie, the "remote" is a bit out of place (it might be used in go-car to verify that you have a "trustless" CAR).
"Bytes" is really just a "block", there's not going to be some arbitrary non-block call here os maybe "Bock" would be more honest?

Reply:
It'd also be super easy to add this callback to the tests in here - we have the list of `expectedBlocks` and we could just `sizeOf` those to compare to a cumulative total received on this callback.